pootle/apps/pootle_store/models.py

```diff
@@ -198,7 +201,9 @@ class Unit(models.Model, base.TranslationUnit):
     store = models.ForeignKey("pootle_store.Store", db_index=True,
                               on_delete=models.CASCADE)
     index = models.IntegerField(db_index=True)
-    unitid = models.TextField(editable=False)
+    unitid = models.TextField(
+        editable=False,
+        validators=[MaxLengthValidator(4096)])
```

Comment:
Not sure if this is actually necessary. Is it possible to edit it from client side?